CONTRIBUTING.md

```diff
@@ -36,113 +37,86 @@ Some of these guidelines are not followed in the oldest code in the repository,
 1. One statement per line, to make semi-automatic processing and reading of diffs much easier.
 2. Boolean Variables should use `stdbool.h`'s names `true` and `false`, not an integer ([reference](https://github.com/fontforge/fontforge/issues/724)).
 3. `return` statements should be inline with the indentation level they are being put on; don't put them at the left margin as in much of the existing code ([reference](https://github.com/fontforge/fontforge/issues/1208)).
-4. Use POSIX/gnulib APIs in preference to glib, e.g. `strdup` instead of `g_strdup` and `xvasprintf` instead of `g_printf_strdup`. 
-This minimizes the impact of non-standard types and functions.
+4. Use POSIX/gnulib APIs in preference to glib, e.g. `strdup` instead of `g_strdup` and `xvasprintf` instead of `g_printf_strdup`. This minimizes the impact of non-standard types and functions.
+
+Beyond these guidelines, try to follow the style used in the file being modified.
 
 ### People To Ask
 
 Various areas of the codebase have been worked on by different people in recent years, so if you are unfamiliar with the general area you're working in, please feel free to chat with people who have experience in that area:
 
 * Build System: Debian - Frank Trampe (frank-trampe)
-* Build System: OS X (Application bundle, Homebrew) - Dr Ben Martin (monkeyiq)
-* Build System: Windows - Jeremie Tan (jtanx)
-* Feature: Collab - Dr Ben Martin
```

Comment:
Ben only works for money, and we're inactivating the feature, but we ought to leave him in for reference in case somebody wants to contact him.